docs/src/literate-tutorials/dg_heat_equation.jl

```diff
@@ -0,0 +1,397 @@
+    ## Reset to 0
+    fill!(Ki, 0)
+    ## Loop over quadrature points
+    for q_point in 1:getnquadpoints(iv)
+        ## Get the normal to face A
+        normal = getnormal(iv, q_point)
+        ## Get the quadrature weight
+        dΓ = getdetJdV(iv, q_point)
+        ## Loop over test shape functions
+        for i in 1:getnbasefunctions(iv)
+            ## Multiply the jump by the normal, as the definition used in Ferrite doesn't include the normals.
+            δu_jump = shape_value_jump(iv, q_point, i) * normal
+            ∇δu_avg = shape_gradient_average(iv, q_point, i)
+            ## Loop over trial shape functions
+            for j in 1:getnbasefunctions(iv)
+                ## Multiply the jump by the normal, as the definition used in Ferrite doesn't include the normals.
+                u_jump = shape_value_jump(iv, q_point, j) * normal
+                ∇u_avg = shape_gradient_average(iv, q_point, j)
+                ## Add contribution to Ki          
+                Ki[i, j] += -(δu_jump ⋅ ∇u_avg + ∇δu_avg ⋅ u_jump)*dΓ +  μ * (δu_jump ⋅ u_jump) * dΓ
+            end
+        end
+    end
+    return Ki
+end
+
+function assemble_boundary!(fe::Vector, fv::FaceValues)
+    ## Reset to 0
+    fill!(fe, 0)
+    ## Loop over quadrature points
+    for q_point in 1:getnquadpoints(fv)
+        ## Get the normal to face A
+        normal = getnormal(fv, q_point)
+        ## Get the quadrature weight
+        ∂Ω = getdetJdV(fv, q_point)
+        ## Loop over test shape functions
+        for i in 1:getnbasefunctions(fv)
+            ## Multiply the jump by the normal, as the definition used in Ferrite doesn't include the normals.
+            δu = shape_value(fv, q_point, i)
+            fe[i] = normal[2] * δu * ∂Ω
+        end
+    end
+    return fe
+end
+#md nothing # hide
+
+# #### Global assembly
+# We define the function `assemble_global` to loop over the elements and do the global
+# assembly. The function takes our `cellvalues`, `interfacevalues`, the sparse matrix `K`, and our DofHandler
+# as input arguments and returns the assembled global stiffness matrix, and the assembled
+# global force vector. We start by allocating `Ke`, `Ki`, `fe`, and the global force vector `f`.
+# We also create an assembler by using `start_assemble`. The assembler lets us assemble into
+# `K` and `f` efficiently. We then start the loop over all the elements. In each loop
+# iteration we reinitialize `cellvalues` (to update derivatives of shape functions etc.),
+# compute the element contribution with `assemble_element!`, and then assemble into the
+# global `K` and `f` with `assemble!`.
+#
+# !!! note "Notation"
+#     Comparing again with [Introduction to FEM](@ref), `f` and `u` correspond to
+#     $\underline{\hat{f}}$ and $\underline{\hat{u}}$, since they represent the discretized
+#     versions. However, through the code we use `f` and `u` instead to reflect the strong
+#     connection between the weak form and the Ferrite implementation.
```

Comment:
As above, I don't think more than a short note is required in this advanced tutorial. 
```suggestion
# We define the function `assemble_global` to loop over all elements and internal faces (interfaces), as well as the external faces involved in Neumann boundary conditions. 
```